Describe W0022 warning (#5683)
DevExpress/devextreme-documentation: api-reference/50 Common/utils/zz Errors and Warnings/W0022.md

---
---
##### shortDescription
Occurs when the DevExtreme version is pre-release and not suitable for commercial use.

---

> The DevExtreme version you use may contain deficiencies and as such, should not be considered for use or integrated in any mission critical application.

Reviewer: ```suggestion
The DevExtreme version you use may contain deficiencies and should not be considered for use or integrated into any critical application.
```
